packages/flutter/lib/src/widgets/decorated_sliver.dart

```diff
@@ -25,15 +25,28 @@ import 'image.dart';
 ///
 /// Commonly used with [BoxDecoration].
 ///
-/// The [child] is not clipped. To clip a child to the shape of a particular
-/// [ShapeDecoration], consider using a [ClipPath] widget.
```

Comment:
I believe "the child is not clipped" is intended to mean "the child is not clipped by this `DecoratedSliver` widget", but I agree that the wording has some room for improvement 👍